collections-csharp/HashtableInCSharp/HashtableInCSharp/HashtableDemo.cs

```diff
@@ -0,0 +1,88 @@
+﻿using System.Collections;
+
+namespace HashtableInCSharp
+{
+    public class HashtableDemo
+    {
+        public Hashtable CreateHashTable()
```

Comment:
This method currently is not being used anywhere, only returns one of the three hash tables and is not covered by any tests. As I have no access to the article, I don't know if this is code that is being shown in the article as ways in which a HashTable can be initialized, but if so, then I would suggest breaking this into three methods, one for each HashTable, and then adding tests around each one.